### ISSUE TITLE
Why is topydo not running in Bodhi Linux installation?

### ISSUE THREAD

**Issue by RonCam** [bram85/topydo#195]
Topydo is running fine on Linux Mint, with my desktop computer.  On my netbook (Bodhi Linux) the installation goes to completion, but the result does not run.  Here are the steps I took and what I see in terminal:
Installation:
`pip3 install topydo`
This is what I see as a result from this installation
`/home/ron/.local/bin contains two Python scripts:  topydo and watchmedo .`
I open terminal in the _bin_ directory and enter _./topydo,_ with the following result:
```
:~/.local/bin$ ./topydo
Traceback (most recent call last):
  File "./topydo", line 7, in <module>
    from topydo.ui.UILoader import main
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/ui/UILoader.py", line 22, in <module>
    from topydo.ui.cli.CLI import CLIApplication
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/ui/cli/CLI.py", line 21, in <module>
    from topydo.ui.CLIApplicationBase import CLIApplicationBase, error
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/ui/CLIApplicationBase.py", line 148, in <module>
    from topydo.lib import TodoList
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/lib/TodoList.py", line 24, in <module>
    from topydo.lib.TodoListBase import TodoListBase
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/lib/TodoListBase.py", line 24, in <module>
    from topydo.lib import Filter
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/lib/Filter.py", line 21, in <module>
    from topydo.lib.Utils import date_string_to_date
  File "/home/ron/.local/lib/python3.4/site-packages/topydo/lib/Utils.py", line 21, in <module>
    import arrow
ImportError: No module named 'arrow'
```
Could someone please look at the above, give me some idea as to what is happening, and then tell me how to fix it?
Many thanks!

**Comment by RonCam** [comment]
Apparently --
`pip3 install topydo`
is installng the previous version, because I see this in terminal:
```
 pip3 search topydo
topydo                    - A powerful todo.txt application for the console
  INSTALLED: 0.12
  LATEST:    0.13
```
Does this have anything to do with the way topydo is (trying to) run?
Is there a command that installs the current version directly, or does one have to first install the older version and then do an update?

**Comment by RonCam** [comment]
I think I should close this before taking anyone's time unnecessarily.

I went back to @mruwek's previous answer (there are 'old version' similarities between Bodhi on the netbook and Mint 17.3 on the desktop) and so I recycled the same information that worked previously:
```
install prompt_toolkit manually: pip3 install prompt_toolkit
Reinstall topydo with both extras at once: pip3 uninstall topydo && pip3 install --user --upgrade topydo[columns,prompt]
```
As soon as the above completed, I noted _./topydo --version_ worked correctly (shows topydo 0.13) so I assume everything else will be working, as soon as I fix the path.
If not, I'll reopen this thread.  
Thanks again!
